contrib/vcloud/vcloudbenchmarkbase.py

```diff
@@ -0,0 +1,126 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import benchexec.benchexec
+import benchexec.model
+import benchexec.tooladapter
+import benchexec.tools
+import benchexec.util
+
+
+class VcloudBenchmarkBase(benchexec.benchexec.BenchExec):
+    """
+    An extension of BenchExec that supports executing the benchmarks in the
+    VerifierCloud (internal project at https://gitlab.com/sosy-lab/software/verifiercloud/).
+    """
+
+    def create_argument_parser(self):
+        parser = super(VcloudBenchmarkBase, self).create_argument_parser()
+        vcloud_args = parser.add_argument_group("Options for using VerifierCloud")
+        vcloud_args.add_argument(
+            "--vcloud",
+            dest="cloud",
+            action="store_true",
+            help="Use VerifierCloud to execute benchmarks.",
+        )
+
+        vcloud_args.add_argument(
+            "--vcloudMaster",
+            dest="cloudMaster",
+            metavar="HOST",
+            help="Sets the master host of the VerifierCloud instance to be used. If this is a HTTP URL, the web interface is used.",
+        )
+
+        vcloud_args.add_argument(
+            "--vcloudPriority",
+            dest="cloudPriority",
+            metavar="PRIORITY",
+            help="Sets the priority for this benchmark used in the VerifierCloud. Possible values are IDLE, LOW, HIGH, URGENT.",
+        )
+
+        vcloud_args.add_argument(
+            "--vcloudCPUModel",
+            dest="cpu_model",
+            type=str,
+            default=None,
+            metavar="CPU_MODEL",
+            help="Only execute runs in the VerifierCloud on CPU models that contain the given string.",
+        )
+
+        vcloud_args.add_argument(
+            "--vcloudUser",
```

Comment:
This parameter is not used here, and the same is true for several others, please remove them.

Reply:
I would need some help here. I actually don't know which other parameters are not used here.

Reply:
They are explicitly documented as such.

Reply:
done